index.html

```diff
@@ -36,29 +37,32 @@ <h3 class="connect"><a href="">LET’S CONNECT</a></h3>
 	</section>
 
 	<section class="work">
-		<div>
-			<img src="/img/tonic1.svg" alt="">
-			<h2>TONIC</h2>
-			<ul class="work-exp">
-				<li>Canopy</li>
-				<li><img src="/img/circle.svg" alt=""></li>
-				<li>Back End Dev</li>
-				<li><img src="/img/circle.svg" alt=""></li>
-				<li>2015</li>
-			</ul>
-			<p>A daily selection of privately personalized reads; 
-				no accounts or sign-ups required.</p>
-      <ul class="lang">
-				<li><a href="">html</a></li>
-				<li><a href="">css</a></li>
-				<li><a href="">javascript</a></li>
-			</ul>
+		<div id="work_1">
+			<div class="img"><img src="/img/tonic1.svg" alt=""></div>
+			<div class="content_work">
+				<h2>TONIC</h2>
+				<ul class="work-exp">
+					<li>Canopy</li>
+					<li><img src="/img/circle.svg" alt=""></li>
+					<li>Back End Dev</li>
+					<li><img src="/img/circle.svg" alt=""></li>
+					<li>2015</li>
+				</ul>
+				<p>A daily selection of privately personalized reads; 
+					no accounts or sign-ups required.</p>
```

Comment:
- [ ] You have done great work implementing the design close to the Figma. 👏  However, as the previous reviewer suggested, ensure that the `font-size` of the `CANOPY` is slightly greater than the `p` description.